### ISSUE TITLE
Add graceful exit option

### ISSUE THREAD

**Issue by Kamrulhasan12345** [lukeed/uvu#138]
HI,
it would be perfect for us if we could add custom scripts for graceful exiting, as sometimes nor gracefully exiting can cause memory leaks.

**Comment by Kamrulhasan12345** [comment]
Yes, cz I facing node process not exiting issue only with bcrypt.hash async function (sync one is ok).

**Comment by lukeed** [comment]
Reopening as a tracker for potential `--force-exit` CLI flag.
